lib/animina_web/live/story_live.ex

```diff
@@ -3,13 +3,19 @@ defmodule AniminaWeb.StoryLive do
 
   alias Animina.Accounts
   alias Animina.Accounts.Photo
+  alias Animina.GenServers.ProfileViewCredits
   alias Animina.Narratives
   alias Animina.Narratives.Headline
   alias Animina.Narratives.Story
   alias AshPhoenix.Form
+  alias Phoenix.PubSub
 
   @impl true
   def mount(%{"id" => story_id}, %{"language" => language} = _session, socket) do
+    if connected?(socket) do
+      PubSub.subscribe(Animina.PubSub, "credits")
```

Comment:
Shouldn't this subscribe to something like `credits` AND a `user_id`? I only want to get an update for the credits for that user and not for all users of the system or do I?